### ISSUE TITLE
Images shift in pdf if pdf is opened multiple times

### ISSUE THREAD

**Issue by saigach** [bpampuch/pdfmake#389]
Hi!
I've encountered a problem. I've got a button on my page, which calls pdfMake.open() function. The problem is that if the button is pressed for the first time, pdf looks fine. But if you press the button more times, all the images in it appear shifted as if every time some margin appears before them. This problem concerns only images, everything is ok with text columns.

![bug](https://cloud.githubusercontent.com/assets/6723560/9944076/850048d6-5d9e-11e5-93b9-07cded0f6115.jpg)

here is the test file

```
<script src="pdfmake.js"></script>
<script src="vfs_fonts.js"></script>

<script>
    docDef = {
                        pageSize: 'A4',
                        pageOrientation: 'landscape',
                        content: [{
                                columns: [{
                                    image: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQYAAACiCAYAAACjz2NwAAAeYElEQVR4Xu1dz48dx3HuWSsKZAPxk50YTgBbK198SWBSAgLkpLdCbkEi8pBDTlohxxxEnXPQ8uIrqX8gXt0DiEbu2sdTDoG41CGXIICWcZDYCQI+JgZk/eCbfNXd8+ZXz1R3z7w3PftqAILk7kx3dVX119XVVdWZkuf6c+Bycapurs+v/0BlhGNxIBurIWknYQ48WuQqU2cAh7sJUymkJcQBAYaEhLEzUggY6MnVuXp9/c7O+pGGrw0HBBiujSh7BlIAg3nlMayHE1gP60MYuowxjgMCDHF8m9dXdWAowOEdgMPjeQ1EqN0XBwQY9sXpKftpAwNRs7aWg4DDlLJJtG8BhkQFMypZbmAwXWSKLIfzUfuTxmbPAQGG2YvQYwB9wGDAQU4sPNh4SK8IMByCtDlgIB7IicUhaIL3GAUYvFk14xd9gMEMT04sZizmMUkXYBiTm6m25Q8MBTjIiUWqstwTXQIMe2L0pN2EAQORKicWkwps+s4FGKaXwe4pCAcGQ5OcWOxeNon2IMCQqGBGJSsWGAw43MFx5gej0iONJc8BAYbkRTQCgUOAgbqXE4sRhDCvJgQY5iWvOGqHAoPpdQXr4bbkWMSJYG5fCTDMTWIx9I4DDNQzHWcSOFzFkCHfzIcDAgzzkVU8peMBA9EgJxbxkpjNlwIMsxHVAELHBYYCHN6THIsBMkn8UwGGxAU0CnnjA4MhS04sRhFPio0IMKQolbFp2hUwEJ1yYjG2tJJoT4AhCTHsmIhdAoMhXU4sdizCfTcvwLBvjk/R3+6BgUb1WL22vjnF8KTP8TkgwDA+T9NrcT/AoAAMok/pST+KIhFkFNtm9pEAw8wENj25AgzTy2D3FAgw7J7H16wHAYZrJlDncFzA8BxXTXx5pNRL5sqJUR7ZSozCxhQaEWBIQQq7pqEJDP8Lsf/ri0q9/FypV78er3cBhvF4OXFLAgwTC2Av3RfAQBhAgPD0G6bb738lwLAXAcyvEwGG+cksnGIChl8CDH4BUKgaCAIM4bw8kC8EGK67oC8XN9Q/v3ipnlkroTpeAYbrLv3o8QkwRLMu8Q8vFwtQ+C5Cls/UP77kJlaAIXEhTkeeAMN0vN9dz5eLJQDhZ+jgWHciwLA7Xl/TlgUYrpNgyUrYABAydas2LAGG6yTlvYxFgGEvbN5DJ5cLs21QirYQ9UeAYQ8CuF5dCDDMXZ7kXMzVPQxj2TmUOQEDWT031+u5i2Xu9AswzFWCVeciN4Y5AMPl4lj7RXL1UL2+JstHngk5IMAwIfOju246F7mGUgYGAoSNeh9+kVM9jFzdFWDgBLr73wsw7J7H4/XQ5VzkekgRGEqL5w7IL/0iAgycNPfyewGGvbB5hE76nItc86kBQ99YBBg4ae7l9wIMe2HzgE58nItc86kAw+XibXtyctxJsgADJ829/F6AYS9sjugkxLnINT81MBifCJ2c3OBIFR8Dy6G9vCDAsBc2B3ZiPPQX+Kp7ZQ1pcipgiLF2xGIIkezO3hVg2BlrBzRsVlgChnGefQND86QhZBQCDCHc2tm7Agw7Y+2AhucKDJl6GaMuIjDjGCDAEMe3kb8SYBiZoaM0N0dg+G+kdf/ec4pYbIdkhzBFgCGEWzt7V4BhZ6wd0PCcgIEA4RcvKPUF6kf+yecDBm0/FWAYzsMRWhBgGIGJozcxB2Ao6kYSIBSPAMPoqjBVgwIMU3G+r9+UgYEA4d9/SzkrQgkwpKhNUTQJMESxbccfpQgMX2LM/4aakbR16HoEGHasGPtrXoBhf7z27ylFYPgMfoRfwlLoewQY/GWc+JsCDCkKSIDhLEWxHBJNAgwpSluAQYBhYr0UYJhYAM7uBRgEGCbWSwGGiQUgwNDggMQxJKGRAgxJiKFBhFgMYjFMrJcCDBMLQCwGsRhSVEEBhhSlIhaDWAwT66UAw8QCEItBLIYUVVCAIUWpiMUgFsPEeinAMLEAxGIQiyFFFRRgSFEqYjGIxTCxXgowTCwAsRjEYkhRBQUYUpSKWAxiMUyslwIMEwtALAaxGFJUQQGGFKUiFoNYDBPrpQDDxAIQi0EshhRVUIAhRamIxSAWw8R6KcAwsQDEYhCLIUUVFGBIUSpiMYjFMLFeCjBMLICDthhy9Uxl6ts1Hkg9hiQ0UoAhCTE0iDgEi4FA4UjRHZ0PMPpXthwQYEhCIwUYkhDDQQLDe+r19X11ubgFcPhIgCEtRRRgSEsehprrbzE8Ua+tj7esf7RY4d9v6P+LxZCERgowJCGGA7MYMvWOurk+3466CoQCDElopABDEmI4KGCoWwvF0B8tHuOfPxGLIQ2FFGBIQw51Kq7LVoIsg426Xzt5yNUH8C3cabH9k8UdvHdPgCENhRRgSEMO1xEYfg4/wi34S04x2X+2HWCmbmIbQdZBc8zHeO8zAYY0FFKAIQ05XD9gyNSrAIArPbBHC/rbHEm+tu7WOdpO0PHl6+uzFMVySDQJMKQo7flvJT4EAJxuWVtaDQ/x82Unyx8tzgEMVwIM0yulAMP0MmhTMHdgyNRtWAsUuGSey8UCE/4p/lUHjObIP1kYS0Eshsm1UoBhchE4CJg7MLi2C48WDwAOj3snvQBDMtoowJCMKCqEzA0YfmfzufpC/bZ67YsjjMK9XfCZ9HQyodRCLIbplVKAYXoZzH8r8aPf/Jn64hun6odf/2UnMBiwO+v1MXyyuI/v1wIM0yulAMP0Mpg3MHzn+a/UX/3X9/UgzOnDquZ4LEZXWEHV04rmyB8tngI8KM7B+BrkmYwDAgyTsb6n4zltJX745d+pP/+fv9ajoe1ChoxJ18lDGcAkAU4p6lyDJgGGFIU0J2D48a9/rP70//5Fs7HPKni0uMAbS/1eO1fiBiyFS/07yZVIQiMFGJIQQ4OIOQHD3/xnqUMF3SZI6fZ2VM3oRwMA5h1jZby/fVeAIQmNFGBIQgwzBoYf/Oan6i+e/q21GKrhzytM+NvIlTi1E3/RYjUda1ZTrsViSEYbBRiSEUWFkDlZDN98/pX6wed/qH73he+pTf4PGEUJAJk60ScRRa2FJq8FGFLUPrvbS5a0AyZsTsBAYqI4hpeeb9SPvv5WTWoCDLNVYrEYUhSdAANZGfJMyAEBhgmZ39n12MDwTy8p9U2s6vR86/mv1VfZr/S/v/v1Q/Xd/O+9WPBs80fqqxe+o75Wf6y+Ovp9Y2/m31NfZAv1Yv4f6svsD8Ri8OLkLF4SYEhRTPHA8CmGc6VzEo7wN/375nq11yES7eRn2KgboIGyJWn1f9tJg/gY9iqakM4EGEK4ta93/YDhCcih+gUrTMDHeweAUF4YwKBiLPQ3/XlF12aQU4lQTu7lfQGGvbA5sBM3MBAQ0BEgpTMTEJBFMN+HUrFvrteIY6CSbpQ8ZQq5SBxDEjIVYEhCDA0iSmB4qAOBjgAIrnJoKdIeS9Pl4hjbj1t6C1St5RDbnnw3iAMCDIPYt6OPaTWlh1ZUeYQDE3BAgGECpkuXwoHUOSDAkLqEhD7hwAQcEGCYgOnSpXAgdQ4IMKQuIaFPODABBwQYJmC6dCkciOHAcvkmpadnq9XHZzHf0zdo455Smw9Wq9VVXxsCDLEclu+EA3viwHK5xCnV0T1M1lPqMlfqHYDDeUj3tg3cCJY/W60udDsCDBwX5PfCgYQ5sFyenGcqq4WVAxxQOHdzFys/e6QNULgFQ4OA5Umu8ls+34jFkLBCCGnCAWv+n2GillWuKmwBQOhI2CzLVw1uLfI8W8JCuAVQOQYgfOhjKRRtCDCI7gkHEucAbQMwuTHxs59EkAorYXMHVkJ5M5hHI4OA4eTk5I08z4+x/8Ef/dwAQlVKeGVk5tibjTdXWZZd4f1PfUwZD9rlFeHAQXEAjsNTTNgzDNrklfQ/AAR1FuqLiLIYgFwEAG8Bucg8gZkS/8C0IQS8Qgv4s3kIsMD/5REOCAc4DmAeYu4d4U9+A0CxXYhzldEiTPPpAXfqwPXhZTEYQrL3h4JBFzFANjhR4o9guEHK74UDwoEwDvQCQ3HEgZfg1ex8ntnVv0ArsgLoeAXbCnoI1TL6d6f5I8AQJjR5Wziwaw50AoO1Ej7CpG6X/TZUARAoj16bLT5HJscAizv2LPbb1YEJMOxazNK+cCCMA05gsE4OBEN0Gv+fwkpY+gBCswVjhWR0LgtfhbUp1Oa2y2tKPg04LGuWBpyXT1z7J3KEuqgdy9mpPcNZ3SuMthEssrLO1TDGo70baI8AEsdKup1VVwuQB2SxwZGU3kOqIf32UVmM0TqUKcLuPGxU5duW5p+HesN96Gu+05Rxly5cXFw8jB2Pq82mLsb0W9GDLWku+Xb0H61/1JmNpLwBOZeXA1kqWsCwS1CoCqUatIHjlBPXxAAtOL/NlwAS2orUrIxQAQPIrtAOJtaGjm1IYVkrp07vcgkwO0Mbx/i5j1fYm0RYTO9BOAhYaT9kuWXqiK532/OTP7xYXYD34Q/kRpbhPfAcPM5fDeV1V48UqAMZwEoN1YccCxk55XI6tsPfYY9xumf3MSb6O+DIcNvvaRcPMA8ehLdL9OcPMaY1YhP6tvnOgdrF+cJu8RFJubnZXOBqwOChhNg+bMhSiFolGxMNZ7NH1M4rXcBQf18f1fRYMWHCtkc5d8O+Mm8Xih/zreubvvEbcHQHt4zVv7udOGCwSvdZsQXd1TaRX8D05CHLliL9gsGgB5zgMzsy92z2PK7J1ve+iVUw86G/3bBApWZbTVDQXHIszFtgaArUrcDjnh4UwvUBBqLnZHmyAmI7twyGXkLR7BxWAVaqIwgwp1DQToSH0kAQOVkrQdaDoeVN6Pw4Tz8wtMNhx+mVayUWGOoATlYDVrWXud5ifg8Z0ITvc2p3WmIx/RXf+OhhjLXlswhAV16O0Vei3QUKHsDAKiBZC8exRLkEoX0I6uizsYDB1Q63ulNIqWuPxSkOBwygRftNjBO3+VBAWE5bEx3/frH6uNMJXCrhFvRoMtBz1VwJOYUl/tB3+IP+WzRRcBqdIJFpim1bLDCckLVQaz8m6YfjP/3eZ7y7iI9x5S5U6cV4kb34MbY8YY8PMPTpSl9vXaBA33RaDMUE7Ws4NNbalyU0wXYJDEQHL8gN7YOLCedFugcwOP0m1cYLRegTNmh/im/gE+Ez4saYKNakvYK8AZh8n43xOLd75N9BW696MTbgpTHGG9Dd9lVuAsdun7h2aWsES8SGAYRRTs7gIjuz+WUPMLDWAqGK8+QgjLz22yRcKA45Z9iJaRw15WmGzwANMBjLpIvWPudf1zfjAANOJrBf7bcYNHB6AddYE4WAlKJSQ4PO8F3LWij4twurYazxhuowN4EHAIN22nbTE2vFdYMCYzGcPO2JV9B0DtnbhDK+631eIO7TDWqvbz8aI8ixlBJ0rQEMzlgROsrCBD3z9TyPRRPxmXgWAgzmxODoo27wHd9qGGu8ofrJ62GcL453/ocDQ5+lUIJ2e95knEDtx0+gvMehDBz7fV4gfcDQ7bj03cpUxzOVUvbxdCyaDCDRURhvxRX0wFqg469lH30xfN7HeEP1lNfDNIDB7+Smw8fADdIwLRypQpnt8z5Ha5/i9UyaZ10rdopKmSJN/EpntQiJc7B+yAE6yjMWEIYSw+vh9MDgCwp6druOKznmmg9zcn4FB1KEMpx7nxdIuMUQ418gOjm+jb06cryZkiYfa6Ggf0y+TCUDXg+nBQZfoO6TCbYS3Q6j8sO4gfooc8g7vEC6Iih18Ah592vPkJOWqZQyNYuBc+w6eD6a1TCVDHg9jJsv/ITmLXfrl6JtXcVvleMW9L54HoePgfOuG4shbqAhk97nXV4gnaHVraOaoWMaQylDnYscj8agieuj+fvmUTCBbbM+YRsc/E5ZOFqmGC/RxOth3HwZCgwuUCA9J5r7omc7thJ8BN/QScQJ2Pf3HgKhPIhGuLaueVc5+6VAIR0zPyism1dKdQ5arnrGdmzq8Sk6m1768iAli6FtLZhzdj4icVhYb8GDGBmgNiLJXUe6xiZVeehhVH2RIcBAssD0v6xaCoVFHHPMn/lZDPPwMfhMLpPYkwFANiiOGV81ilNKH1qsNRYVJedqn6NpzP29WTnr8S9FrAIXbWrGPdxq4MbrIwMTFq+T67z1ITVgcEc1lsFQHJ+6LAZCTyZzkd/b+Ahh6DucQELbNwVmcsTTh1sPHLN9aRnTGuNoGhMYbIRk1W+zDZkvoif79GqIf8fXYvCVQfGeycDNcWdD/4LB6WGsTGMshi5QqJZFiNELWAxcYpI+lXiMU4mboYwe+30PgbxHdQtsv3C+6NqUNva/mxpMGNpafBBCL883cvgYk7V82glgsUo0tcXQlEUzPwC/pzTld/t4OtRq4GRQBUKTs3JEW7dTbiHEWO4juAu65H489HAvW4mO/AeqCo0aC2ViYAifihHTqUTrMgsXO+Ya+WhWL9o69GVlhjtYY5hd8NXsB3U1q3fnCAxWIbep1TSu5iT3Oa0YOvYYGVjaAVr1C1yaOt9HWzrA8CYqrNXKLjrLIsTwCcBgimpwq+WYZijXVzxS98Yx0F6yt8hGyBhjmN0cF9p4jDRxyu48i+VJ9bsxaPKho20tuH1Q3KIztJDLkPFyDjkLdj0FhLprZMQCXshWwhHq3FkrhedTu1AQhUR7Fp6ISyX1UTTfd3ik7gYGz32v95aJZ3Y3LaXl8OZ9bH1oS0unKYOfMWjyIaIZ+9KVYMcrerilNhYQOnwkraGTD8oVqcnr4W6PK0NAgQbF60WbXl0HgDteMui5u4IbPspI7/AC6Z+M3ArmMom7aOOZ7QMMulZDq66CLz8cFsiqb8sUYhF10eAIte3No+H5FK9XfNuj6EOrMAqvh7sBBopAhnyR8Gguty0eTq48nzqAwTeuehepsyGTgBcIpwj9GYAGGPxuEuaZzQNDyNh93t0HTW1robtepQVztiSfL8/HBkKfbXRH8Z/ecnu72krQIUA9JsdPX3m96AQGHTJ8Bcb3HlvuouBGSPTfCMDQW5fBAoOXR5ln9vUDBtcC4uOU5izSWL0aKgO/rU67Dgmvh7uxGJrA6AuoPJ86gMEgO7+ahkwcnxXOrigogUWl6PkkLc5hxJlU1B8X0OWL9lyOiQ8tvjzyfY9XgGFghTFTZN02itQ3FsFvZfaz1Kq8GDpeT2BoOSBTAAZfUDA63x+S4NL5RpVo76PLVrlpX+VtvkcTDPsmRJ3xnnl+gOxWwplMFbJfK97lAcZ/Ehqr6Yjq+5/H8tFPAfxpasupXcYe4OelB2M7fksZcArP6gNbmt8VazE1MISAgp9e9FgM1mrwum7bHjMRkg7KNyisFN/UZx4YuP0ue0eDd22GcYEhvGKSC0A8+IOovjjwaadWh9Uf9Ax4YutkjmsxcFcSuMc4JTD4WmlhfGKAIQIcKGLww5hVrpr04Wt2eyh+r3+AO5Xw3UZ4bkm8J2GI1dTHa357M97eN3TV8gt4CivkwusDZzH0V53qOobl+x2Pz3VrNi75LIZeZ9ly32hB43PQ+QYUX37lCxAmPDXb3ovpGxrrMcDOhCQ+XiMsH4S3GPyUvNh/h4BSF595mmIVth1hF1P1igNmo0/+2x2qoN1Xq7TPEuVO4vpWZk4PYyue9fk8YiyFQk9iFozO+wyM9eB/C5K9vOW8uF+xmtKqVwvcQYk7+ai4KY6vqmnQ/fcqVCcBP8B8nWXqFH3jvLd80P/b5oqxrgt69Y1FcID6XTzjs/pZ0ARYmgtwCr7on2s+HCGXo0wJ71qdfMHW04rxOnFp8K61D4+t6OXn7PMDVL/x6ivyqLo5Fi/zmJDoo3eh+GddvOUm4b6BgaOH0xF+wWhHrvYCg2XksZ1U20toOULCfu+3UvtEqhX9Gh/Iti4DXaJSqWZTp84UstggacYPFCxPeisih43fvB2yUrra5y0iY92F1Fy0eQXbOw5L/sZHwVIYOB+a3u8rsjJgHYcVfbjCv+kPCpb0FqylG9wpgaoTNHz0MPb41QWcQ0HBZxFzJUmywFBBWlrtkVfRn3wSOil8TGjrtSeUPw1tv+N9KECOMOQcpdn9t0DUFm4dfgsXPSDApW71DKUr9oYhO0lINve4Ks0GgEB7lq/6ipQY/496g0rXN2+VMm3olRjHzOphKP8c4bxO1hGdAGy6fLjm4NYTE7eOQwZkAW6PTgfynwDh3C4QGkBcT4geor0H4PP9kNvWm8AwAiiQXqB6Gc+npl54A0OVUXSaYFKaFXUccPtvtRV95RolECFfoFsQCLxa4bcUePUEytgptP5bqE0KNPpDWzotm6zLRkp0v2rVt1WulGrX9xlNsFc4pY1VgEbsyTPwhzklqtPj6td/nNsbp1tpvt3ypFOA/JTjRyOse5scZP0CpC+kD9x4YSW6dLPQBUqJ1wVaSBdIxzofu+rSO1aW+UNmDI2+/U5wqsAQ6tytz89t2AHHI/tZ/fZw2ipGAUOTKcaZSM+R/buLbXpS0n0FvYLgFUfeEA5cPw7YrRuALz+feo6MAgzXT0QyIuHAYXNAgOGw5S+jFw44OSDAIIohHBAOtDggwCBKIRwQDggwiA4IB4QDPAfEYuB5JG8IBw6OAwIMMxN5EeBDZMfepDSzIQu5E3BAgGECpg/pskhEGhIAM6R/+fYwOCDAMCM52wg8lJzf3Jo6AGZGbBNSIzhwEMBQNb+L2PUy5j5/VsTjV7JAtz8jniI/AnkDOivy0yKUuvi+y5y3+QYLrphNpe0nrryDrq2DidvXsd1exXJcY3D9rKpDHG0Fb8CXGr/o5xX+1sZVHU9TX9GOkweVfrb8d+l6RX7evOS2Zi7diZhns/vkIIBBK5ate1ckbZVx6WV2Z5HW3TTTi7TVIgOymnXYlfxkiq/QxL14tUOJKd+EEp+Oi9/bRB66S3Obx1GNny/6MqCQIevR/6bs9hi21Yta5d8pJwEwSLRts1Jt1mHt2jY9adSRvr+ymR1aVHxqJsk1E4WqvOlKqCu+6Uu4A82gV1Fil35snRBKuWZ4afIKOvJG3m+mZw/Ngp0LQhwMMNBkghJfFrcfQUD0/4uiqEZZsLSdBl6dVPiO0oa3qcguYKgWAXH5Aow1Ya4sL+omFoVu8X6t2EwTGEpQoEnrl7JugPFNNG0msB0D9X/cntDbosCUvERWyRrvrSghqW+SU6oxWqcakOtq8dfub3QNDDuRdY0OTE53IZkSZNx3UBSJX0WdiFLWipL0bheT0cVL0gn8fnshb/nuthaJTsWmZCuqy2mTrrystLmAgIvOgwEGGnxZLclcfU5p3Eah6W7LnJQzsxeC4mflU5lUULIMq0iZxuoGhhO61xGT310DwQVCLsvA0FzWqSQQsaBkV/IYYKD04hxl9LNlE4RMf8UKWgJUmXFZ76+5+lN7mDhYgfVk00830FUtNTMJXcDg6KNVMq+oXm0sLlN3wcrzuCqfOi8prVvLnMCRLAvoQPlU2uysClZ9/7r9+8CAoTDBtYmMlNTtPRr63y7zkwReAgMqROlVPv+wqEvRBIbCWjCpq7ooCNqtV1N2TTQ/YND9A7zyFf5G4ZxwYAA9SF9XrxjLKcO2pV4ctrnlMmCxvd+0ViwXNOuCNXasZF1Qu1dmsqm7NNmbNLq3cH3AYOoyEsCgPVwC3L55HfKh7QLJ70OMyQKDmarVoisNkCn4gPd1laeaFVC2GV7W/jqAxMEAQ8N8J/MSk4y2Eqa0XMWUr5mfVWCgf9PWAKsilL/81r3S6NVdF7Zp7l8rE+qq8EFUth+1PX9DmXVtAnude2vS9SlkYytBY9erugO09CXH1WpPpRVRLwFWmvC6ChbAyvDEWg4Pqlu1grYQYKi8q0vvFZciOfwZ6Ct7y7FtOa46dNtbCQN41W1QSeeJbVPXNN36KsivElrLY45AcUDAYAqaFpPUsd/c3sbVVLxyUpnVg1/dTWGOYq9rJ2DtDsRyZSbFoz18TiBCPoeaWVvpq1qwxF6RFm4xlA7UYpWul3sjxa/4FM4NgGqnXus25Sow0MpswfWG9U3UfDhxwLDd1pBlc78CULiH5OK00ib1BR6S1bDdTmC7hPqfAP1232WdUZeFRO9b2dk2dYEg+gOrqF5Dco6T3ofmgwAGshag7KTk5K0+pVXECD6DU4mU5GJplIFWkBzgkeHW64+3Hm5SHvwMxTOM2e361n6PCZvD7M2phqTesxplVjCty5+Viqq9/1QJC4BAitcu0FH0RY668ljVVEJq0tkncDMB9Ph1Oy6eVCYQtlpH1AdOTugbXemIxkSTY/sYK0fToXlDoEKTh2IsSrqpME95y1j585LHzXYsjwmgNA9BM8VtwKlZyqz4WYXmY9BM8qMtDVktLX66ZU4+nOwM79fotDRQmzTGZR8f+vg+19/9P8i8PUDOjO+mAAAAAElFTkSuQmCC'
                                }]
                            }

                        ]
                    };

</script>
<button onclick="pdfMake.createPdf(docDef).open();">Open</button>
```

**Comment by st33bz** [comment]
I encountered the same problem and identified the behavior, if not the cause. The .open() method applies x and y coordinates to image objects in the content or column arrays built. The x value appears to increment itself each time .open() executes, however, rather than just being set once or remeasuring from 0 every time. The result seems to be that images in some circumstances move across the page by the width of the left margin every time the object updates.
I worked around this temporarily by using the .resetXY() method that I stumbled into when picking apart the object.

In your structure identified above, the syntax would be something like:
docDef.content[0].columns[0].resetXY();
fired after you call .open().

It will require someone with much more familiarity with the library than I have to correct the underlying problem, though.

**Comment by depiction** [comment]
I wish this could be fixed soon. st33bz's hack doesn't work in my situation because the PDF object varies.

**Comment by depiction** [comment]
The fix mentioned by @gabucito at https://github.com/bpampuch/pdfmake/issues/355 resolves the problem.

**Comment by liborm85** [comment]
Fix is included in the new version (0.1.22).
